pkg/process/util/containers.go

```diff
@@ -191,15 +191,17 @@ func computeContainerStats(hostCPUCount float64, inStats *metrics.ContainerStats
 		outPreviousStats.SystemCPU = statValue(inStats.CPU.System, -1)
 
 		outStats.CpuLimit = float32(statValue(inStats.CPU.Limit, 0))
-		outStats.TotalPct = float32(cpuRateValue(outPreviousStats.TotalCPU, previousStats.TotalCPU, hostCPUCount, inStats.Timestamp, previousStats.ContainerStatsTimestamp))
-		outStats.UserPct = float32(cpuRateValue(outPreviousStats.UserCPU, previousStats.UserCPU, hostCPUCount, inStats.Timestamp, previousStats.ContainerStatsTimestamp))
-		outStats.SystemPct = float32(cpuRateValue(outPreviousStats.SystemCPU, previousStats.SystemCPU, hostCPUCount, inStats.Timestamp, previousStats.ContainerStatsTimestamp))
+		outStats.TotalPct = float32(cpuRatePctValue(outPreviousStats.TotalCPU, previousStats.TotalCPU, hostCPUCount, inStats.Timestamp, previousStats.ContainerStatsTimestamp))
+		outStats.UserPct = float32(cpuRatePctValue(outPreviousStats.UserCPU, previousStats.UserCPU, hostCPUCount, inStats.Timestamp, previousStats.ContainerStatsTimestamp))
+		outStats.SystemPct = float32(cpuRatePctValue(outPreviousStats.SystemCPU, previousStats.SystemCPU, hostCPUCount, inStats.Timestamp, previousStats.ContainerStatsTimestamp))
+		outStats.CpuUsageNs = float32(cpuRateValue(outPreviousStats.TotalCPU, previousStats.TotalCPU, inStats.Timestamp, previousStats.ContainerStatsTimestamp))
 	}
 
 	if inStats.Memory != nil {
 		outStats.MemoryLimit = uint64(statValue(inStats.Memory.Limit, 0))
 		outStats.MemCache = uint64(statValue(inStats.Memory.Cache, 0))
-		outStats.MemRss = uint64(statValue(inStats.Memory.UsageTotal, 0))
+		outStats.MemRss = uint64(statValue(inStats.Memory.RSS, 0))
```

Comment:
To confirm, this will affect the RSS when we display this metric for a given container and the user will potentially see a large change from an agent upgrade.

Reply:
> Before refactoring (7.39) we were sending RSS in MemRSS (but that was meaningless for Windows). After refactoring we were sending MemoryUsage (for consistency, while app still displays RSS)
> As we have added the MemoryUsage field, we have reverted this change (but it means we'll loose display for Windows if merged before changes are done in app).

Yes, for Linux it will go back to pre-7.39 values and for Windows it will go empty. My idea would be to change the app to check first `MemRss` and fallback to `Usage` if not present or put 2 columns in app, but it must be done before `7.45` release.